person.rb

```diff
@@ -1,8 +1,11 @@
 require './nameable'
+require './date'
+require './rental'
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative or require, and when it is some dependency on locating in gem kindly use require `ex: require 'date'.` so here above I suggest you use require_relative instead of required  for rental `ex: require_relative  './rental'`  and for  date file use require  ex: require 'date' it will fix the error you are facing when running your main.rb file  :+1: :+1: :+1: